.github/workflows/release.yml

```diff
@@ -6,15 +6,24 @@ on:
       - "stable"
       - "rc"
       - "v*.*.*"
-  schedule:
-    - cron: "0 6 * * *"
   workflow_dispatch:
+    inputs:
+      is_nightly:
+        description: "Is this a nightly version or a release - Y or N"
+        type: string
```

Comment:
why not boolean?

Reply:
upd, dq: why revert is_nightly check?

Reply:
OK, it works. the doc seems to be outdated